### ISSUE TITLE
Restoring from seed does not scan old addresses automatically

### ISSUE THREAD

**Issue by dabura667** [darkwallet/darkwallet#102]
I just recently attempted restoring from seed on testnet.

I let it sit there and do it's crypto thing for like 5-10 minutes as I pretty much forgot about it.

I open it up and my balance is 2 BTC when I had 4.17.

Clicking create change address on my spending pocket about six times revealed numerous addresses with 1 tx each and no balance, then finally my 2.17 BTC change address.

Restoration seems like it might be having problems. (These were not stealth, just change)

**Comment by sembrestels** [comment]
Yes, we have to automate the generation of addresses on restoring.

**Comment by caedesvvv** [comment]
At the moment people have to create both pockets, addresses and change addresses to re-discover funds.

**Comment by caedesvvv** [comment]
We have (just) added a tool that scans for pockets and addresses, we will soon fully automatize it.

**Comment by Dark2014** [comment]
@caedesvvv 
The tool to scan is a nice feature, but to seperate it from "Receive Stealth" is not optimal.
I scanned about 15 times and try to "fix" my missing payments, in the end I get success with "Receive Stealth".

So after restore from seed is a fully automatize scan for normal and stealth addresses needed.
Also there should be a checkbox in the "Seed Scanning" options to include the stealth in the scan (this makes it clear that the normal scan is without stealth).
